src/export/digitalPlanning/model.ts

```diff
@@ -31,30 +28,46 @@ export class DigitalPlanning {
   }
 
   /**
-   * Dummy data to test setup whilst we work on the actual schema
+   * WIP
    */
   mapPassportToPayload(): Payload {
     return {
       data: {
-        applicant: {
-          name: "Test",
+        user: {
+          role: this.passport.data?.["user.role"][0],
         },
+        applicant:
+          this.passport.data?.["user.role"][0] === "applicant"
+            ? this.getApplicant()
+            : this.getApplicantWithAgent(),
         property: {
-          name: "Test",
+          address: this.getPropertyAddress(),
+          type: {
+            value: this.passport.data?._address?.["planx_value"],
+            description: this.passport.data?._address?.["planx_description"],
+          },
+          boundary: this.getBoundary(),
+          constraints: {
+            planning: [],
+          },
         },
         application: {
-          name: "Test",
+          type: this.getApplicationType(),
+          fee: this.getApplicationFee(),
+          declaration: {
```

Comment:
I appreciate this is the wrong time / place to notice this but presumably we'd need a timestamp here also somewhere?